src/Payments/BankDebitModal.res

```diff
@@ -317,7 +317,16 @@ let make = (~setModalData) => {
               value=accountType
               setValue=setAccountType
               disabled=false
-              options=["Savings", "Checking"]
+              options=[
+                {
+                  label: "Savings",
+                  value: "Savings",
```

Comment:
Instead of passing same value in both label and value. Could you please also make label as optionalField and if it is not there then infer from value

Reply:
Updated